content/20.pharmaceuticals.md

```diff
@@ -26,29 +27,30 @@ Previous emergent viral threats had a much lower death toll that was not felt on
 For example, SARS-CoV-1 was controlled within 9 months of its appearance, whereas SARS-CoV-2 remains a significant global threat almost 1.5 years later.
 ]({{csse_deaths_figure}} "Global COVID-19 deaths"){#fig:csse-deaths secno=1}
 
-SARS-CoV-2's rapid shift from an unknown virus to a significant worldwide threat initially closely paralleled the emergence of _Severe acute respiratory syndrome-related coronavirus_ (SARS-CoV-1), which was responsible for the 2002 SARS pandemic.
+SARS-CoV-2's rapid shift from an unknown virus to a significant worldwide threat initially closely paralleled the emergence of _Severe acute respiratory syndrome-related coronavirus_ (SARS-CoV-1), which was responsible for the 2002 SARS epidemic.
```

Comment:
```suggestion
SARS-CoV-2's rapid shift from an unknown virus to a significant worldwide threat initially closely paralleled the emergence of _severe acute respiratory syndrome-related coronavirus_ (SARS-CoV-1), which was responsible for the 2002 SARS epidemic.
```

Reply:
This is their formatting thing, they want the first word capitalized! I think, at least.

Reply:
Sorry you totally pointed this out before haha